docs/developer/decision-records/2022-03-30-cosmosdb-lease-mechanism/README.md

```diff
@@ -0,0 +1,36 @@
+# CosmosDb lease mechanism
```

Comment:
Use consistent naming and spelling for Cosmos DB

Reply:
Changed